projects/dashboards/gfbio/src/app/gfbio-collection/gfbio-collection-dialog.component.html

```diff
@@ -1,26 +1,24 @@
 <geoengine-dialog-header>VAT - The Visualization, Analysis and Transformation System</geoengine-dialog-header>
 <mat-dialog-content>
-    <h2>Basket Import</h2>
-    <p>Here is a summary of the imported datasets from your basket!</p>
+    <h2>GFBio Collection Import</h2>
+    <p>Here is a summary of the imported datasets from your collection!</p>
 
     <div class="entry-container">
-        <mat-table #table [dataSource]="result.basket.content">
-            <!-- Name Column -->
-            <ng-container matColumnDef="title">
+        <mat-table #table [dataSource]="collection.items">
+            <ng-container matColumnDef="name">
                 <mat-header-cell *matHeaderCellDef>Dataset</mat-header-cell>
-                <mat-cell *matCellDef="let element"> {{ element.title }} </mat-cell>
+                <mat-cell *matCellDef="let element"> {{ element.name }} </mat-cell>
             </ng-container>
 
-            <!-- Statius Column -->
             <ng-container matColumnDef="status">
                 <mat-header-cell *matHeaderCellDef>Status </mat-header-cell>
                 <mat-cell *matCellDef="let element"
-                    ><span class="{{ element.status }}">{{ element.status }}</span></mat-cell
+                    ><span class="{{ layerStatus(element) }}">{{ layerStatus(element) }}</span></mat-cell
                 >
             </ng-container>
 
-            <mat-header-row *matHeaderRowDef="['title', 'status']"></mat-header-row>
-            <mat-row *matRowDef="let row; columns: ['title', 'status']"></mat-row>
+            <mat-header-row *matHeaderRowDef="['name', 'status']"></mat-header-row>
```

Comment:
it doesn't really add any functionality because users can just remove the layer once it was added. If we have time/capacity we can do it after everything else works. Otherwise we can do it in a future project.

Reply:
Ok, let's keep this open and decide upon everything else is resolved.